### ISSUE TITLE
Bundler could not find compatible versions for gem "rails"

### ISSUE THREAD

**Issue by scryptclub** [owen2345/camaleon-cms-language-editor#1]
I used rails 5.
When I try to install the editor, I got the follwing error:

```
Bundler could not find compatible versions for gem "rails":
  In Gemfile:
    rails (~> 5.0.1)

    camaleon_cms (>= 2.4.2) was resolved to 2.4.2, which depends on
      cama_contact_form (~> 0.0.16) was resolved to 0.0.18, which depends on
        rails

    cama_language_editor (>= 1.1) was resolved to 1.1, which depends on
      rails (~> 4.2.1)
```

**Comment by owen2345** [comment]
Fixed, please use 1.2

**Comment by scryptclub** [comment]
It is all working now. Thank you very much.
